common/file_reader.h

```diff
@@ -410,6 +410,17 @@ class FileReader {
     itoa(v, value, 10);
     write_key_value(key, value);
   }
+  void write_key_value_float(const char* key, float f) {
+    int  a_ = (int)floorf(f);
```

Comment:
variables with underscores are meant for class-level variables.
Also, we don't need variables for this as all as we can just inline the floorf() call in the itoa below.

Reply:
I have made changes